### PR TITLE
Lighthouse on CI - Warnings only

### LINKED ISSUE #13266: Add lighthouse check on CI
Get the lighthouse score/report for each PR

### PR DESCRIPTION
**Pull request by hackergil**
Fixes #13266

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -108,6 +108,7 @@
     "@angular/pwa": "^12.0.5",
     "@angular/router": "^12.0.5",
     "@angular/service-worker": "^12.0.5",
+    "@lhci/cli": "^0.8.1",
```
Comment (marlass) [inline package.json]: I see it also installed globally in action? Should it be also in our package.json? If so then I guess in devDependencies

File: .travis.yml [hunk 2]
```diff
@@ -2,6 +2,8 @@ dist: bionic
 language: node_js
 node_js:
   - 12.16.1
+before_install:
+  - npm install -g @lhci/cli@0.8.x
```
Comment (marlass) [inline .travis.yml]: Can we move it to lhci.sh instead? So it's only installed for one task (not all of them)

Comment (hackergil) [reply .travis.yml]: yep

File: package.json (hunk 1 above)
Comment (hackergil) [reply package.json]: nope. Removing

**Review by marlass — APPROVED** [review]

**Comment by hackergil** [comment]
Closing in favor of #13710

**Pull request closed without merging**